Dockerfile

```diff
@@ -16,6 +16,8 @@ WORKDIR /
 COPY --from=builder /build/views /views
 COPY --from=builder /build/public /public
 COPY --from=builder /build/go-test-bench /go-test-bench
+# Copy /etc/passwd for easy path-traversal
+COPY --from=builder /etc/passwd /etc/passwd
```

Comment:
I know we're intentionally vulnerable and all but I think we can just cat some stuff into `/etc/passwd` in the image and demonstrate the vulnerability just as well. Better than having docker copy potentially sensitive stuff into the image when it builds it.

Reply:
I'm also happy for us to just change the default form value in the front-end to something which is more likely to be there, but less important files may not demonstrate the risk quite as well.

Reply:
Yeah, that was my original thought. Then I realized we're in a container, copying the file from a prebuilt image, so I'm not sure how sensitive that really is. What do you think about filling it with fake content, but still conforming to the standard `/etc/passwd` format? I'm thinking something like:

```
root:x:0:0:root:/root:/bin/bash
catty:x:42:42:boh:/etc/contrastsecurity:/bin/bash
```

Reply:
That works. Let's be sure to add a comment about why we're doing it because it'll look kinda odd on its own.

If we want, this can also be moved into an `init()` in the path-traversal vuln package (or similar)